pkgs/development/libraries/graphene/default.nix

```diff
@@ -62,6 +62,7 @@ stdenv.mkDerivation rec {
 
   postPatch = ''
     patchShebangs tests/gen-installed-test.py
+    patchShebangs tests/introspection.py
```

Comment:
And gi is actually from pygobject so something like `PATH=${python3.withPackages (pp: [ pp.pygobject3 ])}/bin:$PATH patchShebangs tests/introspection.py`

Reply:
Actually, it's not the only missing dependency, it also require a module called tap which I guess comes from the tappy lib.

Reply:
Your snippet (with tappy added) works, unfortunately the test fails. It tries to load graphene: `gi.require_version('Graphene', '1.0')`, I'm not sure how we could reference the derivation itself in it's definition.

BTW, while trying things around I realized that this test is also skipped in the check phase (same reason, pygobject3 not available). Maybe keeping this test skipped is acceptable for the moment?

Reply:
You will also need to wrap the test to set `GI_TYPELIB_PATH`, see the gjs package for example.

Reply:
Seems to finally work. Thanks for your help :)